modules/floating-buttons/base/widget-floating-button-base.php

```diff
@@ -1298,21 +1416,40 @@ protected function add_style_chat_button_section(): void {
 			);
 		}
 
+		if ( $config['style']['chat_button_section']['has_drop_shadow'] ) {
+			$this->add_group_control(
+				Group_Control_Box_Shadow::get_type(),
+				[
+					'name' => 'style_chat_button_box_shadow',
+					'fields_options' => [
+						'box_shadow_type' => [
+							'default' => 'yes',
+						],
+						'box_shadow' => [
+							'default' => [
+								'horizontal' => 4,
+								'vertical' => 4,
+								'blur' => 10,
+								'spread' => 0,
+								'color' => 'rgba(0,0,0,0.15)',
+							],
+							'selectors' => [
+								'{{WRAPPER}} .e-contact-buttons__chat-button-drop-shadow' => 'filter: drop-shadow({{HORIZONTAL}}px {{VERTICAL}}px {{BLUR}}px {{COLOR}});',
```

Comment:
@louiswol94 changing CSS classes is a lot of effort and could be really disruptive, that's why I have not done it in this PR. If we feel it is needed, it will be a separate PR